ihmc-common-walking-control-modules/src/main/java/us/ihmc/commonWalkingControlModules/capturePoint/BalanceManager.java

```diff
@@ -455,10 +465,10 @@ public boolean checkAndUpdateStepAdjustment(Footstep footstep)
    {
       boolean usingStepAdjustment = stepAdjustmentController.useStepAdjustment();
 
-      if (!usingStepAdjustment)
-      {
-         return false;
-      }
+//      if (!usingStepAdjustment)
+//      {
+//         return false;
+//      }
```

Comment:
Commenting this out allows us to view the capture region, and the boolean got pushed further down.